rust/Cargo.toml

```diff
@@ -1,11 +1,12 @@
-# Software Name: its-client
-# SPDX-FileCopyrightText: Copyright (c) 2016-2023 Orange
-# SPDX-License-Identifier: MIT License
+# Software Name : libits-client
+# SPDX-FileCopyrightText: Copyright (c) Orange SA
+# SPDX-License-Identifier: MIT
 #
-# This software is distributed under the MIT license, see LICENSE.txt file for more details.
+# This software is distributed under the MIT license,
+# see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
 #
-# Author: Nicolas Buffon <nicolas.buffon@orange.com> et al.
-# Software description: This Intelligent Transportation Systems (ITS) [MQTT](https://mqtt.org/) client based on the [JSon](https://www.json.org) [ETSI](https://www.etsi.org/committee/its) specification transcription provides a ready to connect project for the mobility (connected and autonomous vehicles, road side units, vulnerable road users,...).
+# Authors: see CONTRIBUTORS.md
+# Software description: This Intelligent Transportation Systems (ITS) [MQTT](https://mqtt.org/) library based on the [JSon](https://www.json.org) [ETSI](https://www.etsi.org/committee/its) specification transcription provides a ready to connect project for the mobility (connected and autonomous vehicles, road side units, vulnerable road users,...).
```

Comment:
You removed the description from all the source files; should we also just remove it from here too?

Reply:
Forget it, GitHub presents commits in an usual order...

Reply:
Yes, I always wonder which commit I have to start with when I start a review